src/components/AssignmentSummary.jsx

```diff
@@ -226,13 +229,15 @@ export class AssignmentSummaryBase extends Component {
                 color: "secondary",
                 hideBadge: true
               })}
-              {window.NOT_IN_USA && window.ALLOW_SEND_ALL && !hasPopupSidebox
+              {this.props.assignment.campaign.organization.allowSendAll &&
+              campaignAllowBulkSend &&
+              !hasPopupSidebox
                 ? this.renderBadgedButton({
                     assignment,
                     title: "Send messages",
                     primary: true,
                     disabled: false,
-                    contactsFilter: "all",
+                    contactsFilter: "text",
```

Comment:
When `contactsFilter` is "all", the "Send messages" button displays all the time. When `contactsFilter` is "text", the "Send messages" button displays only when there are available contacts to text. This change makes it so that the "Send messages" button isn't always displaying for bulk send even if there are no available contacts to text.